### ISSUE TITLE
Unable to resume music with headphones hardware button

### ISSUE THREAD

**Issue by martin-desktops** [SimpleMobileTools/Simple-Music-Player#414]
Though pausing music is flawless, I am unable to resume music using a hardware button on my headphones 

**Conditions:**
Headphones: JBL Tune 500BT
OS: Android 11
App: Simple Music Player 5.11.3

**Steps to reproduce:**
- Play music on your headphones
- Pause music using hardware button: as in result, music is paused
- Try to resume music using the same button: as of result, the music remains stopped

**Expected behaviour (this is how it works in AntennaPod, Spotify, VLC…):** 
Pressing the button again resumes music

**Possibly related with:**
 https://github.com/SimpleMobileTools/Simple-Music-Player/issues/395

**Comment by tibbi** [comment]
does double pressing skip to the next track?

**Comment by tibbi** [comment]
ok, if I send you a test version, can you try it out? You would have to uninstall your current version. You will see some message show up on button press.

**Comment by martin-desktops** [comment]
Of course!

**Comment by tibbi** [comment]
ok, here it is. To install the test version first uninstall your current one and allow installing apps from unknown sources in the security settings of your device. Once you download the attachment on your device just remove the ".zip" extension by renaming. Github doesn't allow sending apk files directly for security reasons. After renaming just run it, it should be straightforward. If you cannot find the item related to installing apps from unknown sources, go into the device Settings -> Apps -> Special app access -> Install unknown apps and select your file manager, or the app you use for launching the sent file, then enable the "Allow from this source" item.
[music-player-core-release.apk.zip](https://github.com/SimpleMobileTools/Simple-Music-Player/files/8947926/music-player-core-release.apk.zip)

**Comment by tibbi** [comment]
what messages do you see when pausing and when resume should happen?

**Comment by martin-desktops** [comment]
On pausing, I see "media 127" message, but on resuming it doesn't either resume music or show any message.

**Comment by tibbi** [comment]
[music-player-core-release.apk.zip](https://github.com/SimpleMobileTools/Simple-Music-Player/files/8948082/music-player-core-release.apk.zip)
how about this? If you still see nothing at resuming, does it work well in other apps? Isnt your system swallowing the action for some reason?

**Comment by martin-desktops** [comment]
On pausing, the app says "action android.intent.action.MEDIA_BUTTON", but on resuming there is no toast notification again.
I am not sure about my system, though it works that way, correctly, in the following apps: VLC, Transistor, ViMusic, AntennaPod, Vinyl, Howl, LibreTube.

**Comment by tibbi** [comment]
try this one please. If it wont work, Ill take a look at it some other day, it might be some bigger issue, not just the button itself.
[music-player-core-release.apk.zip](https://github.com/SimpleMobileTools/Simple-Music-Player/files/8948992/music-player-core-release.apk.zip)

**Comment by martin-desktops** [comment]
Unfortunately the latest build doesn't fix the issue. Thanks for today's struggling

**Comment by salazarbarrera** [comment]
Just an addition. This used to be my main player, but I had to move to other one because apparently Simple Music Player is invisible regarding media control. Things I've noticed:

- Last.FM scrobblers based on notification inspection don't detect Simple Music Player
- Sending media key presses through adb doesn't work (probably this is why headphones can't control the player). E.g. Trying to stop reproduction with `$adb shell input keyevent 127`
- KDEConnect doesn't detect the player either. 

Months ago I tried to fix this myself by looking at your code and comparing it to [Music Player GO](https://github.com/enricocid/Music-Player-GO), but I couldn't find any meaningful difference. I don't know why this player is "invisible" and others aren't.
